.gitignore

```diff
@@ -58,3 +58,7 @@ typings/
 .env
 
 dist
+
+# IntelliJ Idea Project files
```

Comment:
I think such things should be handled in global .gitignore instead of this library.

Reply:
I think @timhagn uses IntelliJ and they wanted to ignore .idea files :)

Reply:
*g* I usually have em in a .gitignore_global - maybe I've got to review my settings for the .ignore plugin ^^. Will remove it.